templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Confidential Computing{% endblock %}
+{% block meta_description %}Protect data in use with confidential computing. Build the foundation of your
+privacy-enhancing technology strategy with Ubuntu confidential VMs on both public and private clouds.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1QGwS5CYlwXkaPibSFvcag6zLcUH5lNDUbM18hCqbOJM/edit{% endblock %}
+
+{% block outer_content %}
+<div class="is-paper">
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-section">
+          <h1 class="u-no-margin--bottom">Ubuntu for confidential computing</h1>
+          <p class="p-heading--2">
+            Protect your data in use,
+            <br class="u-hide--small u-hide--medium" />across public and private clouds.
+          </p>
+        </div>
+        <hr />
+        <a class="js-invoke-modal p-button--positive" href="/contact-us">Contact us</a>
+      </div>
+    </div>
+
+    <div class="u-fixed-width u-no-padding">
+      <img src="https://assets.ubuntu.com/v1/7f34ade9-website_suru_25.jpg" alt=""
+        style="aspect-ratio: 5.177570093457944; width: 100%" />
+    </div>
+  </section>
+
+  <section class="p-strip">
```

Comment:
```suggestion
  <section class="p-section">
```

Reply:
in Figma the distance between the Suru image and the separator has to be 64px. But with this change it becomes 0.

Reply:
64 but at the bottom only.